docs/ref/django-admin.txt

```diff
@@ -1156,6 +1162,12 @@ Suppresses all user prompts.
 Sets the name of the squashed migration. When omitted, the name is based on the
 first and last migration, with ``_squashed_`` in between.
 
+.. django-admin-option:: --no-header
+
+.. versionadded:: 2.2
+
+Does not add a header comment to the squashed migration file.
```

Comment:
"Generate squashed migration file without Django version and timestamp header."

Reply:
Done